pkg/controller/volume/persistentvolume/index.go

```diff
@@ -207,6 +217,9 @@ func findMatchingVolume(
 		if v1helper.GetPersistentVolumeClass(volume) != requestedClass {
 			continue
 		}
+		if !nodeAffinityValid {
```

Comment:
If the code runs here, `nodeAffinityValid` will always be `true` since the value has not been changed from L172?